0.19.0-release-notes.md

```diff
@@ -32,3 +32,7 @@ and compiled, even if it is not executed:
 
 -   A new type of interactive abbreviation: `edit:command-abbr`
     ([#1472](https://b.elv.sh/1472)).
+
+-   The `compare` command now supports comparing booleans
+    ([#1585](https://b.elv.sh/1585)). This also means that the `order` command
+    can now sort booleans.
```

Comment:
Better to just say that both `compare` and `order` both support boolean values now.